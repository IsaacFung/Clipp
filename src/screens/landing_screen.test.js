import React from "react";
import renderer from 'react-test-renderer';

import LandingScreen from './landing_screen'

it('renders correctly', () => {
    const tree = renderer.create(<LandingScreen/>).toJSON();
    expect(tree).toMatchSnapshot();
});