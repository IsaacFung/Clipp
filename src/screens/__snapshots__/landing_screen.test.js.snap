// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "margin": "auto",
      }
    }
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#65CE88",
        "borderRadius": 4,
        "margin": 16,
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 20,
          "fontWeight": "bold",
          "paddingBottom": 16,
          "paddingEnd": 32,
          "paddingStart": 32,
          "paddingTop": 16,
          "textTransform": "uppercase",
        }
      }
    >
      Log In
    </Text>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#65CE88",
        "borderRadius": 4,
        "margin": 16,
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 20,
          "fontWeight": "bold",
          "paddingBottom": 16,
          "paddingEnd": 32,
          "paddingStart": 32,
          "paddingTop": 16,
          "textTransform": "uppercase",
        }
      }
    >
      Sign Up
    </Text>
  </View>
</View>
`;
